BP/common/culture/cultures/00_central_germanic.txt

```diff
@@ -244,6 +244,7 @@
 			{ name = "mercenary_company_saxon_steelbearers" }
 			{ name = "mercenary_company_hansa_guard" }
 			{ name = "mercenary_company_sons_of_widukind" }
+			{ name = "mercenary_company_veterans_of_the_marches" }
```

Comment:
If this refers to the border marches of the Carolingian Empire, it is anachronistic at start and shouldn't be active, unless it refers to Roman _limes_?

Reply:
Removed them